src/app/components/AddressForm.jsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Button, Grid, Stack, TextField, Tooltip } from '@mui/material';
```

Comment:
Dividir partes do componente em partes menores; adicionar também o campo de descrição quando a [tarefa do backend for concluída](tinyurl.com/48awc3zj); melhorar nomes dos campos e variáveis utilizadas.

Reply:
@coutocouto dá uma testada nisso por favor.